### PR TITLE
PharoObjectModel - revise Class methods and Class instance vars sections

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Addresses issue #40.

### CODE REVIEW

File: PharoObjectModel/PharoObjectModel.pier [hunk 1]
```diff
@@ -386,11 +402,29 @@ think that they were the same. And indeed, until someone creates a subclass
 of ==WebServer==, they are the same. But suppose that ==ReliableWebServer== is
 a subclass of ==WebServer==, and inherits the ==uniqueInstance== method. We
 would clearly expect ==ReliableWebServer uniqueInstance== to answer a
-==ReliableWebServer==:. Using ""self"" ensures that this will happen, since it
+==ReliableWebServer==. Using ==self== ensures that this will happen, since it
 will be bound to the respective class. Note also that ==WebServer== and
 ==ReliableWebServer== will each have their own class instance variable called
 ==uniqueInstance==. These two variables will of course have different values.
 
+!!!!A note on lazy initialization
```
Comment (dmitrizagidulin) [inline PharoObjectModel/PharoObjectModel.pier]: Added a note on lazy initialization

**Comment by dmitrizagidulin** [comment]
@Ducasse - take a look at this PR when you have a chance.

**Comment by Ducasse** [comment]
I pushed a better text because this is not that reliableWeb has another variable it gets its own value.

**Comment by dmitrizagidulin** [comment]
@Ducasse - wait, what about the rest of the edits to this section that were in this PR? :)

Do you disagree with them, or discarded by accident?

**Comment by Ducasse** [comment]
By accident I did not see the rest.

**Comment by dmitrizagidulin** [comment]
No problem, I'll make another PR incorporating your recent commit.

**Pull request closed without merging**